contentcuration/contentcuration/static/js/edit_channel/models.js

```diff
@@ -276,6 +280,9 @@ var ChannelCollection = BaseCollection.extend({
 	list_name:"channel-list",
     getName:function(){
 		return "ChannelCollection";
+	},
+	comparator:function(channel){
+		return -new Date(channel.get('main_tree').created);
```

Comment:
When a user adds a channel, it gets added to the top of the page. However, when they refresh, it jumps to the bottom, which seemed a little off